redis/nox.py

```diff
@@ -13,97 +15,40 @@
 # limitations under the License.
 
 from __future__ import absolute_import
-
 import os
 
 import nox
 
 
-LOCAL_DEPS = (
-    os.path.join('..', 'api_core'),
-    os.path.join('..', 'core'),
-)
-
-
```

Comment:
Losing the change (PR #5865) to install dependencies "in-place" is not desirable:  likely this means a change in synthtool.  In the meanwhilde, I'd say we back out this all the changes to this file, except maybe adding `3.7` to the tested Python versions.